### ISSUE TITLE
Add non-modal download form

### ISSUE THREAD

**Issue by ghost1227** [easydigitaldownloads/edd-free-downloads#13]
https://secure.helpscout.net/conversation/97137128/10757

**Comment by pippinsplugins** [comment]
:+1:

**Comment by ghost1227** [comment]
Just for my clarification when I get around to this issue, the idea here is to add an option to set a 'free download page' with a shortcode that outputs the form rather than a modal.

**Comment by pippinsplugins** [comment]
Yep, that should work fine. Kind of like a separate checkout form.

**Comment by pippinsplugins** [comment]
New request for this https://secure.helpscout.net/conversation/435769792/67126/?folderId=180499
